modules/ModuleBuilder/parsers/parser.label.php

```diff
@@ -197,25 +197,19 @@ public static function removeLabel($language, $label, $labelvalue, $moduleName,
      */
     public static function addLabels($language, $labels, $moduleName, $basepath = null, $forRelationshipLabel = false)
     {
-        $GLOBALS [ 'log' ]->debug("ParserLabel::addLabels($language, \$labels, $moduleName, $basepath );");
-        $GLOBALS [ 'log' ]->debug('$labels:'.print_r($labels, true));
+        $GLOBALS ['log']->debug("ParserLabel::addLabels($language, \$labels, $moduleName, $basepath );");
```

Comment:
Could we use LoggerManager here instead of the superglobal?

Reply:
Yes will clean this up more soon, it's originally copied from the ModuleInstall::merge_files method.